tabbycat/breakqual/aida.py

```diff
@@ -130,8 +130,16 @@ def reinsert_capped_teams(self):
 
 
 @register
-class Aida2016AustralsBreakGenerator(BaseAida2016BreakGenerator):
-    key = "aida-2016-australs"
+class Aida2016Australs3CapBreakGenerator(BaseAida2016BreakGenerator):
+    key = "aida-2016-australs-3cap"
```

Comment:
`institution_cap = 3` here?

Reply:
The base AIDA generator has the `institution_cap` set at 3 by default, so I only included an override variable in the new option (so the 3-team one should still inherit `institution_cap` from the base)